Share/NCShareExtension.swift

```diff
@@ -65,6 +66,9 @@ class NCShareExtension: UIViewController, NCListCellDelegate, NCEmptyDataSetDele
     private var activeAccount: tableAccount!
     private let chunckSize = CCUtility.getChunkSize() * 1000000
     
+    private var numberFilesName: Int = 0
```

Comment:
Is this being used for something else or is there another reason you can't just do:
```suggestion
    private var numberFilesName: Int { filesName.count }
```